addons/mail/views/mail_activity_views.xml

```diff
@@ -295,19 +295,24 @@
         <field name="model">mail.activity</field>
         <field name="arch" type="xml">
             <tree string="Next Activities"
-                    decoration-danger="date_deadline &lt; current_date"
-                    decoration-warning="date_deadline == current_date"
-                    decoration-success="date_deadline &gt; current_date"
+                    decoration-danger="date_deadline &lt; current_date and active == True"
+                    decoration-warning="date_deadline == current_date and active == True"
+                    decoration-success="date_deadline &gt; current_date and active == True"
                     default_order="date_deadline" create="false">
+                <header>
+                    <button name="action_done" type="object" string="Done" icon="fa-check"/>
+                    <button name="unlink" type="object" string="Cancel" icon="fa-times"/>
+                    <button name="action_snooze" type="object" string="Snooze 7d" icon="fa-bell-slash"/>
```

Comment:
Since we are using the same view, we can't really avoid these buttons, I have made sure that these buttons do not perform anything incase done filtered content is shown. I may be missing something here, let me know if there is any way possible.